packages/file_selector/file_selector/lib/file_selector.dart

```diff
@@ -10,6 +10,18 @@ export 'package:file_selector_platform_interface/file_selector_platform_interfac
     show XFile, XTypeGroup;
 
 /// Open file dialog for loading files and return a file path
+///
+/// The [acceptedTypeGroups] argument is the file type that can be selected in the dialog.
```

Comment:
You don't need to say that an argument is an argument. In all of these, you can replace "The [foo] argument" with just "[foo]".